File: src/main/java/cn/chahuyun/economy/model/currency/GoldEconomyCurrency.java
package cn.chahuyun.economy.model.currency;

import cn.chahuyun.economy.constant.Constant;
import xyz.cssxsh.mirai.economy.service.EconomyCurrency;

/**
 * 金币货币模型
 */
public class GoldEconomyCurrency extends EconomyCurrency {

    public GoldEconomyCurrency() {
        super(Constant.CURRENCY_GOLD_CODE, Constant.CURRENCY_GOLD_NAME);
    }
}

File: src/main/kotlin/cn/chahuyun/economy/model/currency/GoldEconomyCurrency.kt
package cn.chahuyun.economy.model.currency

import xyz.cssxsh.mirai.economy.service.EconomyCurrency

/**
 * 金币货币模型
 */
class GoldEconomyCurrency : EconomyCurrency {
    override val id: String = "hy-gold"
    override val name: String = "金币"
    override val description: String = "壶言经济金币"
}

